modules/software-catalogs/proc-searching-and-filter-software-catalogs.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-about-software-catalogs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-searching-and-filtering-software-catalogs_{context}"]
+= Searching and Filtering Software Catalogs
+
+You can search and filter Software Catalogs by its *Kind* or by using the *Filter* field.
+
+== Searching and Filtering Software Catalogs by Kind
+
+To filter a Software Catalog by its *Kind*, complete the following steps:
+
+.Procedure
+
+. In your {product} navigation menu, click *Catalog*.
+. On the *Catalog* page, click the *Kind* drop-down list.
+. Select the type of *Kind* you want to filter.
++
+[NOTE]
+====
+The available filter dropdowns vary based on the *Kind* you select, displaying options relevant to that specific entity type.
+====
+
+== Searching and Filtering Software Catalogs using the Filter field
```

Comment:
Second level titles are not allowed in procedures. Split in 2 procedures and form assemblies.